docs/content/getting-started/configuration-overview.md

```diff
@@ -93,3 +93,12 @@ All available environment variables can be found [here](../reference/static-conf
 All the configuration options are documented in their related section.
 
 You can browse the available features in the menu, the [providers](../providers/overview.md), or the [routing section](../routing/overview.md) to see them in action.
+
+!!! question "Using Traefik for Business Applications?"
+
+If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/). You can use it as your:
+- [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
+- [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
+- [API Gateway](https://traefik.io/solutions/api-gateway/)
+
+Traefik Enterprise enables centralized access management, distributed Let's Encrypt, and other advanced capabilities. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

Comment:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
    You can use it as your:

    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    - [API Gateway](https://traefik.io/solutions/api-gateway/)

    Traefik Enterprise enables centralized access management,
    distributed Let's Encrypt,
    and other advanced capabilities.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

This change goes from :

![Screenshot from 2022-07-04 09-27-12](https://user-images.githubusercontent.com/10183798/177104125-8ccdda17-36c4-4ada-8390-8f3168b31f7a.png)

to:

![Screenshot from 2022-07-04 09-27-22](https://user-images.githubusercontent.com/10183798/177104095-252d4de3-034e-4ef0-9a55-519d243f0c1e.png)

Next time, feel free to test your changes on the documentation by doing `make docs-serve` :stuck_out_tongue: